dataland-backend/src/test/kotlin/org/dataland/datalandbackend/services/DataOwnersManagerTest.kt

```diff
@@ -24,26 +24,30 @@ class DataOwnersManagerTest {
     lateinit var mockDataOwnersRepository: DataOwnerRepository
     lateinit var mockCompanyRepository: StoredCompanyRepository
 
+    private val testUserId = UUID.randomUUID().toString()
+    private val mockAuthentication = AuthenticationMock.mockJwtAuthentication(
+        "username",
+        testUserId,
+        setOf(DatalandRealmRole.ROLE_USER),
+    )
+
     @BeforeEach
```

Comment:
Wieso eig. nicht nur "Before".
Es reicht doch wenn man das alles einmal initialisiert wird, oder?

Reply:
Das wiederholte aufrufen von mock dient hier als reset, damit die when/then Änderungen aus den einzelnen Tests wieder gelöscht werden. Alternativ könnten wir das in einen before block packen und dann aber in einem beforeeach block nur mockito.reset verwenden. Was meinst du?